src/material/button/_button-base.scss

```diff
@@ -50,6 +50,8 @@
   // The focus indicator should match the bounds of the entire button.
   .mat-mdc-focus-indicator {
     @include layout-common.fill();
+
+    pointer-events: none;
```

Comment:
This one should be applied to all the focus indicators here: https://github.com/angular/components/blob/main/src/material/core/focus-indicators/_private.scss#L15

Reply:
I updated pr, I moved these changes to this file

Reply:
Okay, I see. It's because the focus indicator isn't placed directly on the element like all the other usages.